vnext/Microsoft.ReactNative/Fabric/Composition/CompositionContextHelper.cpp

```diff
@@ -1043,6 +1043,7 @@ struct CompCaretVisual
     // TODO should make the caret use an invert brush by default
     m_compVisual.Brush(m_compositor.CreateColorBrush({255, 0, 0, 0} /* Black */));
     m_compVisual.Opacity(1.0f);
+    m_isVisible = true;
```

Comment:
On line 1098 this is initialized to false. Should that be updated instead?

Reply:
ahhh good catch! That makes more sense than updating it here :)